bower.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "uber",
+    "version": "14.0.0",
+    "homepage": "https://github.com/magfest/ubersystem",
+    "authors": [
+        "Eli Courtwright <eli@courtwright.org>"
+    ],
+    "description": "MAGFest Ubersystem",
+    "main": "uber/templates/base.html",
+    "license": "BSD",
+    "ignore": [
+        "**/.*",
+        "node_modules",
+        "bower_components",
+        "test",
+        "tests"
+    ],
```

Comment:
This is where we specify our dependencies.

Note that I've deleted all of the old dependencies under the ``uber/static/lib`` directory, since this is how those are managed now.